src/device-registry/config/constants.js

```diff
@@ -1,11 +1,12 @@
 const mongoose = require("mongoose");
 const ObjectId = mongoose.Types.ObjectId;
-
 const {
   generateDateFormatWithoutHrs,
   monthsInfront,
 } = require("../utils/date");
 
+
```

Comment:
I guess there is no real change here. Whenever you have such cases where the change is just about code formatting, you can always undo them before **committing** and **pushing**. @BenjaminSsempala 

This ensures that the number of changed files are the actual ones.

Reply:
Thank you @Baalmart, let me resolve those.

Reply:
@Baalmart 
For the circular dependencies, I was using the console output to find them and the exact functions, but the tool seems to be more straightforward.